src/components/cluster/detail/scale_cluster_modal.js

```diff
@@ -88,38 +93,68 @@ class ScaleClusterModal extends React.Component {
         maxValid: nodeCountSelector.scaling.maxValid,
       },
     });
-  };
 
   submit = () => {
     this.setState(
       {
         loading: true,
       },
       () => {
-        var scaling = {
-          min: this.state.scaling.min,
-          max: this.state.scaling.max,
-        };
-
-        this.props.clusterActions
-          .clusterPatch(this.props.cluster, { scaling: scaling })
-          .then(patchedCluster => {
-            this.close();
-
-            new FlashMessage(
-              'The cluster will be scaled within the next couple of minutes.',
-              messageType.SUCCESS,
-              messageTTL.SHORT
-            );
-
-            this.props.clusterActions.clusterLoadDetailsSuccess(patchedCluster);
-          })
-          .catch(error => {
-            this.setState({
-              loading: false,
-              error: error,
+        if (this.state.nodePool) {
+          // Node Pools
+          const scaling = {
+            Min: this.state.scaling.min,
```

Comment:
Is this supposed to be capitalized?

Reply:
I can store `Min` property as `min` but the ideal solution is to have `min` fetched as `min` instead as `Min`,  and `max intead of `Max`.

ping @marians

Reply:
Not sure I understand what's going on here.. so nodepools store their min/max as Min and Max?

Oh! I see it now. These are the parameters for the scaling action. Hmm I think there might be something wrong in the api spec then, or api implementation?

Reply:
Is this still not fixed in the API? The JSON attributes have to be lowercase of course. Let's get this fixed ASAP.

Reply:
See https://github.com/giantswarm/api/pull/842

Reply:
I can approve it if you want... but can't judge the code

Reply:
Thanks for the quickness! :-)

Reply:
https://github.com/giantswarm/api/pull/842 is merged